requirements.txt

```diff
@@ -1 +1,5 @@
-requests>=2.20.0
\ No newline at end of file
+certifi==2022.9.24 ; python_version >= "3.7" and python_version < "4"
```

Comment:
just out of curiosity: how do you generate dependency info so specifically? :D

Reply:
some poetry magic, e.g. `poetry export --without-hashes --only main -o requirements.txt`